src/verinfast/dependencies/walkers/nuget.py

```diff
@@ -18,32 +18,27 @@ def initialize(self, command: str = None):
             if command:
                 return super().initialize(command)
         except Exception as e:
-            self.log(
-                f"NuGet API not found for: {discoveryUrl}",
-                display=False
-            )
+            self.log(f"NuGet API not found for: {discoveryUrl}", display=False)
             self.log(e, display=False)
             return None
 
     def get_license(self, name: str, version: str) -> str:
         resp = None
         name2 = name.lower()
         try:
-            license_resp = self.getUrl(f"{self.registrationUrl}{name}/{version}.json")  # NOQA:E501
+            license_resp = self.getUrl(
+                f"{self.registrationUrl}{name}/{version}.json"
+            )  # NOQA:E501
```

Comment:
This is really repetitive. I struggled a lot to spell that. Can you tell I started this review at the bottom?